code/create_db2.sql

```diff
@@ -0,0 +1,33 @@
+drop table if EXISTS rotten_tomatoes_movies;
+
+-- Búðu til töfluna fyrir CSV gögnin
+CREATE TABLE IF NOT EXISTS rotten_tomatoes_movies (
+    rotten_tomatoes_link TEXT,
+    movie_title TEXT,
+    movie_info TEXT,
+    critics_consensus TEXT,
+    content_rating TEXT,
+    genres TEXT,
+    directors TEXT,
+    authors TEXT,
+    actors TEXT,
+    original_release_date TEXT,
+    streaming_release_date TEXT,
+    runtime INTEGER,
+    production_company TEXT,
+    tomatometer_status TEXT,
+    tomatometer_rating REAL,
```

Comment:
Það mætti bæta við skilyrðum fyrir töluleg gögn, t.d. að þau mega ekki vera neikvæð:
```suggestion
    tomatometer_rating REAL CHECK(tomatometer_rating >= 0),
```